src/Controls/src/Core/Page/Page.cs

```diff
@@ -583,29 +582,6 @@ protected void UpdateChildrenLayout()
 			}
 		}
 
-		internal virtual void OnChildMeasureInvalidated(VisualElement child, InvalidationTrigger trigger)
-		{
-			var container = this as IPageContainer<Page>;
-			if (container != null)
-			{
-				Page page = container.CurrentPage;
-				if (page != null && page.IsVisible && (!page.IsPlatformEnabled || !page.IsPlatformStateConsistent))
-					return;
-			}
-			else
-			{
-				var logicalChildren = this.InternalChildren;
-				for (var i = 0; i < logicalChildren.Count; i++)
-				{
-					var v = logicalChildren[i] as VisualElement;
-					if (v != null && v.IsVisible && (!v.IsPlatformEnabled || !v.IsPlatformStateConsistent))
-						return;
-				}
-			}
-
-			InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged);
```

Comment:
This was already not happening due to the fact there was no `MeasureInvalidated` propagation prior #23052

Reply:
No :-/ I've been wanting to get rid of this code

The main edge case I could see is if someone was using shimmed pages on iOS but in theory those should still follow the maui layout system via the shim wrapper